.github/workflows/pull_request.yml

```diff
@@ -99,6 +99,8 @@ jobs:
         EQ_RUN_FUNCTIONAL_TESTS_HEADLESS: True
       steps:
         - uses: actions/checkout@v2
+        - name: Update to latest stable Chrome
+          run: sudo apt-get update && sudo apt-get upgrade google-chrome-stable -y
```

Comment:
Given our functional test run against the version of Chrome installed on the host machines, it causes issues when developing/CI.

Functional test on master won't run, if you have updated your Chrome to version 83.
In this PR I have updated chrome driver to 83, but actions does not have the latest Chrome yet, only version 81 which is incompatible with Chromedrver 83. Therefore, we need to think of a way to ensure parity between dev and CI, whether it is using a docker image etc.

Reply:
Chrome is now installed from source, so the added time is negligible now which removes the gap between actions being out of date with the latest Chrome.